docs/QuickStart.md

```diff
@@ -237,15 +276,8 @@ POST to ```v1/subjects/USA/child``` to create a
     ]
 }
 ```
-### Coming Soon: Auth Tokens
-
-Refocus will soon also allow you to use auth tokens to add an additional layer of security on the platform. Stay tuned!
-
-### Create your own lens
-//TODO point at ldk
-
-# Thank You!
-Congrats on the deployment of your Refocus! If you have any questions or feedback, please reach out to us.
+### Authorization Tokens Are Coming Soon!
 
+Soon you can use authorization tokens to add an additional layer of security on the platform. Stay tuned!
```

Comment:
@iamigo I'll update the section with your changes and, @meynet I'll remove the authorization info from here.